openlibrary/templates/observations/review_component.html

```diff
@@ -0,0 +1,36 @@
+$def with(work, reader_observations, key)
+
+$if reader_observations['total_respondents']:
+  $ total_reviews = reader_observations['total_respondents']
+$else:
+  $ total_reviews = 0
+
+<a name="reader-observations" class="section-anchor"></a>      
+
+<div class="tab-section review-component">
+  <h2 class="observation-title">$_('Community Reviews') ($total_reviews)</h2>
+  $if total_reviews > 0:
+    <div class="reviews">
+    $for o in reader_observations['observations']:
+      $ total = o['total_respondents_for_type']
+        <span class="label" title="$o['description']">$o['label'] ($total)</span>
```

Comment:
This classname is too generic; it's conflicting with other things on the page. Let's use BEM:

```html
        <span class="reviews__label" title="$o['description']">$o['label'] ($total)</span>
```

Reply:
Can we place these all in a wrapper span? That'll make some styling possible.

Eg

```html
<span class="reviews__group">
    <span class="reviews__label">...</span>
    $for ...:
        <span class="reviews__pill">...</span>
```

Reply:
Something like this would be nice; that'll let us style this as well. You can add the parenthesis using CSS.

```html
        <span class="label" title="$o['description']">$o['label'] <span class="reviews__numeric">$total</span></span>
```

Reply:
> Can we place these all in a wrapper span? That'll make some styling possible.
Will likely need some help getting everything to wrap properly once this change is made.